benchmarks/pvc/flash_attention_v2/benchmark_runner.hpp

```diff
@@ -389,10 +389,10 @@ template <class FMHAConfiguration> struct BenchmarkRunnerFMHA {
     stride_V = cutlass::make_cute_packed_stride(StrideV{}, cute::make_shape(head_size_vo, seq_len_kv, batch * num_heads));
     stride_O = cutlass::make_cute_packed_stride(StrideO{}, cute::make_shape(seq_len_qo, head_size_vo, batch * num_heads));
 
-    auto mem_size_q = batch * num_heads * seq_len_qo * head_size_qk;
-    auto mem_size_k = batch * num_heads * seq_len_kv * head_size_qk;
-    auto mem_size_v = batch * num_heads * seq_len_kv * head_size_vo;
-    auto mem_size_o = batch * num_heads * seq_len_qo * head_size_vo;
+    std::size_t mem_size_q = batch * num_heads * seq_len_qo * head_size_qk;
+    std::size_t mem_size_k = batch * num_heads * seq_len_kv * head_size_qk;
+    std::size_t mem_size_v = batch * num_heads * seq_len_kv * head_size_vo;
+    std::size_t mem_size_o = batch * num_heads * seq_len_qo * head_size_vo;
```

Comment:
you need to cast the first factor into larger size, not just result after it has already overflown.

Reply:
Done - cheers